### PR TITLE
[backend/frontend] Add EE identity support

### LINKED ISSUE #4765: Display a "By Filigran" logo on login page and on all pages of the app
## Use case

Ability for our comminutiy users to clearly see that Filigran developped the tool and for Entreprise Edition users to remove it, in order to easily create a whitelabled solution
## Proposed Solution

Add a Filigran logo on the login page and on all pages of the app.
Create a toogle within settings to remove this logo (only avaiable to Entreprise Edition users).

## If the feature request is approved, would you be willing to submit a PR?

Yes / No (Help can be provided if you need assistance submitting a PR)

### PR DESCRIPTION
**Pull request by Kedae**

### CODE REVIEW

File: opencti-platform/opencti-front/jsconfig.json [hunk 1]
```diff
@@ -1,7 +0,0 @@
-{
```
Comment (SarahBocognano) [inline opencti-platform/opencti-front/jsconfig.json:1]: Why did you delete this file ?

**Review by SarahBocognano — COMMENTED** [review]
Capture d'écran 2023-11-03 110416

Maybe you can close the border ? I find it weird that this is open

File: opencti-platform/opencti-front/jsconfig.json (hunk 1 above)
Comment (Kedae) [reply opencti-platform/opencti-front/jsconfig.json:1]: Because it is not working and confusing :D

**Comment by Kedae** [comment]
> Capture d'écran 2023-11-03 110416
> 
> Maybe you can close the border ? I find it weird that this is open

I can, but it is actually the same border as above separating the items ? @nino-filigran a thought ?

**Comment by Kedae** [comment]
Localization not done, waiting for validation on semantic

**Comment by nino-filigran** [comment]
At first, I was not bothered by it. But indeed, after a 2nd thought, having it closed might be better, I agree with you @Kedae.
From a scalability point of view, if we are to add more toggles specific to EE, then we would also have this "closed" and dedicated space, which is why I'm even more in favor of your suggestion.

Unless also @Jipegien you would like something different?

File: opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx [hunk 2]
```diff
@@ -502,6 +503,31 @@ const LeftBar = () => {
           </Security>
         </MenuList>
       </Security>
+      {!platform_whitemark && (
+        <MenuItem
+          dense={true}
+          style={{ marginBottom: bannerHeightNumber }}
+          classes={{
+            root: navOpen ? classes.menuLogoOpen : classes.menuLogo,
+          }}
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx:518]: This menu item is clickable but does nothing. Open the filigran website in a new window ?

```suggestion
          }}
          onClick={() => window.open('https://filigran.io/', '_blank')}
```

**Comment by labo-flg** [comment]
I suggest to make these 2 buttons consistent in the UI (same position)

image

image

**Comment by Kedae** [comment]
> I suggest to make these 2 buttons consistent in the UI (same position)
> 
> image
> 
> image

Ah oui tiens, j'ai cassé.

**Comment by labo-flg** [comment]
How far this PR should go in terms of identifying the EE features inside the app ?
We have this nice chip, tooltips and specific color, but what should happen if EE is enabled?

For instance, the "share with organization" button is still in EE color, but the automation (playbooks) screen does not display any clear labelling or color or chip to tell the user he's using some EE cool feature :)

**Comment by Kedae** [comment]
> How far this PR should go in terms of identifying the EE features inside the app ? We have this nice chip, tooltips and specific color, but what should happen if EE is enabled?
> 
> For instance, the "share with organization" button is still in EE color, but the automation (playbooks) screen does not display any clear labelling or color or chip to tell the user he's using some EE cool feature :)

That's a mistake, every 'EE' UI item should disappear when under EE.

**Comment by nino-filigran** [comment]
280243469-b93458be-086a-40ae-a3bf-4c800da8ace4


After talking with  @Jipegien, we agreed that the divider bar in grey + the violet bar is a bit weird.
Therefore, is it possible to do as the image above? (sorry it's ugly but want to ensure it's clear. The blue bar in the image is simply there to show that the violet one should not exist)
- have the divider bar directly in violet and remove the existing violet upper bar? 
- do not close the space between the 2 bars

**Comment by Kedae** [comment]
Every comment as been taken care of (still missing Localization)

File: opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx (hunk 2 above)
Comment (labo-flg) [reply opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx:518]: @Kedae wdyt?

**Comment by yassine-ouaamou** [comment]
Since the organization admin feature is EE, it would be better to block this part also:
image

**Comment by Kedae** [comment]
> Since the organization admin feature is EE, it would be better to block this part also: image

Good spot, will do.

**Comment by yassine-ouaamou** [comment]
In the Activity section, I couldn't click the agree checkbox when I click on "Enable EE" from the pop-up. It works fine from the other one.
Here are some shots to make it clear:
Working:
ee_agree_bug_working_path

Not working:
ee_agree_bug

**Comment by Archidoit** [comment]
The 'share' button seems to be present in the overview of every entity... Except for entities of type Data Sources 
image

**Comment by Archidoit** [comment]
In the pop-up to activate EE, the 'associated blog post' link redirects without opening a new tab and we 'loose' the openCTI page (whereas a new tab is openend when we click on the other links).

image

**Comment by yassine-ouaamou** [comment]
@Kedae Can you update the branch with master so I can do another test and approve this PR, please?
Merci

**Comment by yassine-ouaamou** [comment]
ee_activitiy_clicked
ee_file_indexing_clic
File indexing should be in purple as Activity

**Review by yassine-ouaamou — APPROVED** [review]
